### PR TITLE
PIA-986: Dashboard Connection Details Tests

### PR DESCRIPTION
**Pull request by xv-geneva-parayno**
Added Dashboard Connection Details Tests; Optimised elements and methods on test screen classes

### CODE REVIEW

File: PIA VPN/Settings/SettingPopoverSelectionView.swift [hunk 1]
```diff
@@ -313,7 +314,7 @@ extension HandshakePopoverSelectionView: UITableViewDelegate, UITableViewDataSou
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        
+        self.accessibilityIdentifier = "HandshakePopoverSelectionView"
```
Comment (kp-laura-sempere) [inline PIA VPN/Settings/SettingPopoverSelectionView.swift:317]: Nice 👍

File: PIA-VPN_E2E_Tests/Screens/EditDashboardScreen.swift [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        cell(with: "UsageTileCollectionViewCell")
+    }
+    
+    var favoriteServersTileCell: XCUIElement {
+        cell(with: "FavoriteServersTileCollectionViewCell")
+    }
+    
+    var connectionTileCell: XCUIElement {
+        cell(with: "ConnectionTileCollectionViewCell")
+    }
```
Comment (kp-laura-sempere) [inline PIA-VPN_E2E_Tests/Screens/EditDashboardScreen.swift:34]: Great call adding the `accesibilityIdentifiers` in all those elements 👍

**Review by kp-laura-sempere — APPROVED** [review]
Great job @xv-geneva-parayno !
Good call adding the `accesibilityIdentifiers` in the cells 👍

**Review by kp-said-rehouni — APPROVED** [review]
LGTM! Great job @xv-geneva-parayno!

**Pull request merged**